lib/puppet/type/incron_system_table.rb

```diff
@@ -44,7 +44,10 @@
         'IN_NO_LOOP',
         'IN_ONESHOT',
         'IN_ONLYDIR',
-        'IN_OPEN'
+        'IN_OPEN',
```

Comment:
- Missing IN_MOVED_TO for latest version
- IN_NO_LOOP does not exist in latest version

Reply:
@lnemsick-simp I think the omission of `IN_NO_LOOP` in the latest version is a mistake. At any rate, presence of the option did not cause any issues.

Added `IN_MOVED_TO` in a later PR, thanks!